src/components/VideoCardHome/VideoCardHome.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext } from 'react';
+// import { Image } from 'semantic-ui-react';
+import { Card, CardContent, CardMedia, Typography } from '@material-ui/core';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+
+import moment from 'moment';
+
+import VideosContext from '../../context/VideosContext';
+import ThemeContext from '../../context/ThemeContext';
+
+const darkTheme = createMuiTheme({
+  palette: {
+    type: 'dark',
+  },
+
+  transitions: {
+    duration: {
+      standard: 500,
+    },
+    easing: {
+      easeIn: 'cubic-bezier(0.4, 0, 1, 1)',
+    },
+  },
+});
+
+const lightTheme = createMuiTheme({
+  transitions: {
+    duration: {
+      standard: 500,
+    },
+    easing: {
+      easeIn: 'cubic-bezier(0.4, 0, 1, 1)',
+    },
+  },
+});
+
+const VideoCardHome = ({ video }) => {
+  const { onVideoSelect } = useContext(VideosContext);
+  const { light } = useContext(ThemeContext);
+
+  return (
+    <ThemeProvider theme={light ? lightTheme : darkTheme}>
```

Comment:
I'm not very aware of this but... If VideoCardHome component is being rendered in the App (you are using here ThemeProvider) component as child you don't need to wrap again using ThemeProvider